### ISSUE TITLE
Monthly VA.gov Dashboard Update

### ISSUE THREAD

**Issue by michelle-dooley** [department-of-veterans-affairs/va.gov-team#41947]
## Issue Description

Per Chris Johnston's request we need to add a new card to the [Monthly VA.gov Summary](https://va-gov.domo.com/page/1314425343) dashboard.  The new card should be for "Unique Logins" similar to what is found on the [Performance Dashboard](https://www.va.gov/performance-dashboard/).  We should also rename the current "Unique Logins" card to "Total Logins" as that more accurately represents that data.

[Slack thread](https://dsva.slack.com/archives/C1Q4294JJ/p1651606155489009) for reference.

---
## Tasks
- [x] Add new "Unique Logins" card (Jonathan)
- [x] Change name of current "Unique Logins" card to "Total Logins" (Jonathan)
- [x] QA data (Chloe)

## Acceptance Criteria
- [x] The new card has been added and the report has been updated with proper naming conventions.
- [x] The updates are completed for the May report that is distributed 1st week of June.
---
## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by chloedotbrown** [comment]
Documenting recent team conversations on Slack and in stand-up - we'll be removing the event-scoped "Total Logins" card and replacing it with two user-scoped cards:

1. Unique Users (all LOA)
2. Unique Users (LOA3 - identity verified only)

To support these cards, I've created a [new datasource](https://va-gov.domo.com/datasources/d32a7f4b-7916-4194-917f-d45722b2ea62/details/overview) in Domo. Monthly totals match our previous dataset for LOA3.

**Comment by michelle-dooley** [comment]
Great, thanks @chloedotbrown!!  @jonathan-epstein13 is it possible to add an explanation to the card that All LOA included verified and non-verified and LOA 3 only included verified?  Is there anything else we need to do before we sent this?  Also when we do we will want to explain that the performance dashboard is using sampling and that is the the numbers do not exactly match.

**Comment by michelle-dooley** [comment]
We've removed the total logins card (which was showing the total number of logins for the users ie; if a user logged in more than once then it would be counted more than once.)
Added 2 new slides (cards) -->Unique Users (all LOA, identity-verified & non-verified) & Unique Users (LOA 3, identity-verified).
Unique Users (LOA 3, identity-verified) slide/card shows data for identity-verified Veteran logins.
Unique Users (all LOA, identity-verified & non-verified) slide/card shows data for those who signed up/login on VA.gov but have not yet verified identity and those that have completed identity verification.
